pkgs/top-level/all-packages.nix

```diff
@@ -31899,7 +31899,10 @@ with pkgs;
 
   spotify-unwrapped = callPackage ../applications/audio/spotify { };
 
-  spotify = callPackage ../applications/audio/spotify/wrapper.nix { };
+  spotify = if stdenv.isDarwin then
+    spotify-unwrapped
```

Comment:
This should be done one level higher.

Reply:
What level are you suggesting?

Reply:
I mean one level lower.

It is a bit awkward to get a wrapped package on Linux and an unwrapped on darwin from the same attribute. One a second look we could remove that wrapper entirely and move the one functionality into the spotify package itself then we also don't need to make this awkward condition here.

Reply:
It's not really unwrapped, it's an entirely different package. It just happens to be called the same way the unwrapped linux package is and it's simplest to make that attr an alias.